app/javascript/stylesheets/users.scss

```diff
@@ -65,20 +72,25 @@ span.insert-user {
   margin-left: 10px;
 }
 
-.user-avatar  {
+.user-avatar {
   width: auto;
   height: 100px;
 }
+
 .user-avatar-show {
-  margin: 30px 275px 0px 0px;
+  margin: 30px 275px 0 0;
 }
 
-.user-avatar-show .show-user{
+.user-avatar-show .show-user {
   width: auto;
   height: 150px;
 }
 
-.show-box{
+.navbar-user {
+  margin-top: 20px;
+}
+
+.show-box {
   display: flex;
   flex-direction: row-reverse;
```

Comment:
why here row-rewerse opition? in row users is sorted in reverse from right to left?

Reply:
fixed